CONTRIBUTING.md

```diff
@@ -25,8 +25,9 @@ To create the executable in the `target/universal/stage` directory: `sbt stage`.
 
 1. Follow the [sbt-bintray publishing guidelines](https://github.com/sbt/sbt-bintray#publishing) to ensure you are
    authenticated with Bintray.  Note that you need to be a member of the `sky-uk` Bintray organization. 
-2. Run `sbt ciRelease` to perform the release.
-3. Check you are happy with the draft publication of the new version [here](https://bintray.com/sky-uk/oss-maven/kafka-configurator)
+2. Run `docker login --username=sky` and when prompted add the password for the `sky` Docker Hub account.
```

Comment:
yeah was just leaving in sky to speak to you guys about if we want to merge with the `matthedude` placeholder in. Will change tomorrow depending on what ppl want to do.